source/op/pt/comm.cc

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+#ifdef GOOGLE_CUDA
```

Comment:
```suggestion
#ifdef GOOGLE_CUDA || TENSORFLOW_USE_ROCM
```

Reply:
It might be a bit strange to use `GOOGLE_CUDA` and `TENSORFLOW_USE_ROCM` in the pytorch... but it's fine.